### ISSUE TITLE
CleanUpIT fails after #1374

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#1449]
I've been seeing this test failure since #1374 was merged into the master branch:

```java
java.lang.AssertionError: Threads did not go away. Saw 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.accumulo.test.functional.CleanUpIT.run(CleanUpIT.java:148)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by Manno15** [comment]
Looks like Thrift Connection Pool Checker thread does not get properly closed now since #1374. I will work on a fix.
